.github/workflows/infra_drift_detection.yml

```diff
@@ -182,5 +166,22 @@ jobs:
             :wavy_dash: Resource to change: ${{ env.TO_CHANGE }}
             :heavy_minus_sign: Resource to destroy: ${{ env.TO_DESTROY }}
           footer: "Linked Repo <{repo_url}|{repo}>"
+        env:
+          SLACK_WEBHOOK_URL: ${{ env.SLACK_WEBHOOK_URL }}
+
+      - name: Failure Notification
+        if: ${{ always() && (env.SLACK_WEBHOOK_URL || inputs.webhook_url) }}
+        uses: ravsamhq/notify-slack-action@2.3.0
```

Comment:
needs to be pinned to hash, you can use https://github.com/mheap/pin-github-action

Reply:
I'd use https://github.com/slackapi/slack-github-action instead which comes from a verified vendor